index.html

```diff
@@ -8,12 +8,117 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="karma-store" href="/store.html">
 </head>
+
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <header>
+        <img class="svg-logo"
+            src="./img/karma-logo.svg" 
+            alt="Karma company logo">
+        </img>
+        <nav class="nav-bar">
+            <ul class="navigation__list">
+                <li class="active">Meet Karma</li> 
+                <li>How it Works </li>
+                <li><a href="./store.html">Store</a></li>
```

Comment:
well done adding the link here. You lost the styling on this link text now that there is a nested anchor element, can you see how to fix that?